src/plone/restapi/tests/test_dxfield_serializer.py

```diff
@@ -325,6 +325,19 @@ def test_relationlist_field_serialization_returns_list(self):
             value,
         )
 
+    def test_relation_field_serialization_do_not_change_request(self):
+        self.request.form["metadata_fields"] = ["foo", "bar"]
+        doc2 = self.portal[
+            self.portal.invokeFactory(
+                "DXTestDocument",
+                id="doc2",
+                title="Referenceable Document",
+                description="Description 2",
+            )
+        ]
+        self.serialize("test_relationchoice_field", doc2)
+        self.assertEqual(self.request.form["metadata_fields"], ["foo", "bar"])
```

Comment:
It'd be slightly preferable to use real metadata names and check that they are present in the serialized object (in other words, test the result, not the implementation)